backend/src/main/java/com/recipes/backend/bizz/ingredient/IngredientServiceImpl.java

```diff
@@ -27,90 +25,103 @@
 
 @Slf4j
 @Service
-public class IngredientServiceImpl implements IngredientService {
+public class IngredientServiceImpl implements IngredientService
+{
 
     private final IngredientRepository ingredientRepository;
 
     @Autowired
-    public IngredientServiceImpl(final IngredientRepository ingredientRepository) {
+    public IngredientServiceImpl(final IngredientRepository ingredientRepository)
+    {
         this.ingredientRepository = ingredientRepository;
     }
 
     @Override
-    public void addIngredient(final Ingredient ingredient) {
-        final Optional<IngredientDTO> ingredientDTOOpt = IngredientMapper.mapToIngredientDTO(ingredient);
-
-        ingredientDTOOpt.ifPresent(ingredientDTO -> {
-            try {
-                ingredientRepository.save(ingredientDTO);
-            } catch (final DataIntegrityViolationException e) {
-                throw new IngredientDuplicateException(ingredient.getName());
-            } catch (final DataAccessException e) {
-                throw new DatabaseSaveException("couldn't save ingredient " + ingredient.getName());
-            }
-        });
+    public void addIngredient(final Ingredient ingredient)
+    {
+        final IngredientDTO ingredientDTO = IngredientMapper.mapToIngredientDTO(ingredient).orElseThrow(IngredientEmptyException::new);
+
+        try
+        {
+            ingredientRepository.save(ingredientDTO);
+        } catch (final DataIntegrityViolationException e)
+        {
+            throw new IngredientDuplicateException(ingredient.getName());
+        } catch (final DataAccessException e)
+        {
+            throw new DatabaseSaveException("couldn't save ingredient " + ingredient.getName());
+        }
```

Comment:
this formatting looks worse than before, less readable, too many unnecessary empty lines